FluidSynth/FluidSynth.cpp

```diff
@@ -121,14 +122,15 @@ class FluidSynth
         }
     }
 
-    void setOctaveTuning(int chan, Chuck_ArrayFloat * tuning)
+    void setOctaveTuning(int chan, Chuck_ArrayFloat * tuning, CK_DL_API api)
     {
         bool allChans = false;
         if (chan < 0) {
             allChans = true;
             chan = 0;
         }
-        fluid_synth_activate_octave_tuning(m_synth, 0, chan, "", &tuning->m_vector[0], false);
+        fluid_synth_activate_octave_tuning(m_synth, 0, chan, "", (double *) tuning, false);
```

Comment:
Probably need to extract all of the values from the tuning Array and then pass that vector